### PR TITLE
[Reopen] Unify remove to drop in schema 

### PR DESCRIPTION
**Pull request by darionyaphet**
subtask of #348

### CODE REVIEW

File: src/graph/CMakeLists.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ add_library(
     InterimResult.cpp
     VariableHolder.cpp
     AddHostsExecutor.cpp
-    RemoveHostsExecutor.cpp
+    DropHostsExecutor.cpp
```
Comment (dangleptr) [inline src/graph/CMakeLists.txt]: Keep it as it be until we have a conclusion.

Comment (darionyaphet) [reply src/graph/CMakeLists.txt]: I will keep `RemoveHosts` and update if there is progress.

File: src/meta/processors/DropTagProcessor.h [hunk 2]
```diff
@@ -4,29 +4,30 @@
  * attached with Common Clause Condition 1.0, found in the LICENSES directory.
  */
 
-#ifndef NEBULA_GRAPH_REMOVETAGPROCESSOR_H
-#define NEBULA_GRAPH_REMOVETAGPROCESSOR_H
+#ifndef META_REMOVETAGPROCESSOR_H
```
Comment (laura-ding) [inline src/meta/processors/DropTagProcessor.h]: DROP

File: src/meta/processors/DropTagProcessor.h [hunk 3]
```diff
@@ -4,29 +4,30 @@
...
             : BaseProcessor<cpp2::ExecResp>(kvstore) {}
 
     StatusOr<std::vector<std::string>> getTagKeys(GraphSpaceID id, const std::string& tagName);
 };
 
 }  // namespace meta
 }  // namespace nebula
-#endif  // NEBULA_GRAPH_REMOVETAGPROCESSOR_H
+
+#endif  // META_REMOVETAGPROCESSOR_H
```
Comment (laura-ding) [inline src/meta/processors/DropTagProcessor.h]: ditto

File: src/meta/processors/DropTagProcessor.h (hunk 2 above)
Comment (darionyaphet) [reply src/meta/processors/DropTagProcessor.h]: Good catch ~

File: src/meta/processors/DropEdgeProcessor.h [hunk 4]
```diff
@@ -31,3 +31,4 @@ class RemoveEdgeProcessor : public BaseProcessor<cpp2::ExecResp> {
 }  // namespace nebula
 
 #endif  // META_REMOVEEDGEPROCESSOR_H_
```
Comment (laura-ding) [inline src/meta/processors/DropEdgeProcessor.h]: DROP

**Pull request merged**